CRM/Civicase/Hook/PreProcess/CaseCategoryWordReplacementsForNewCase.php

```diff
@@ -47,7 +47,7 @@ private function addWordReplacements(CRM_Core_Form $form, $caseCategoryId) {
     // We need to translate this manually as Civi does not the page title
     // through the ts function.
     $pageTitle = $form->get_template_vars('activityType');
-    CRM_Utils_System::setTitle(ts($pageTitle));
+    CRM_Utils_System::setTitle($pageTitle);
```

Comment:
Removing the translation function here will also break existing functionality.

Reply:
Fixed, created a new function as discussed.